plugins/mcreator-localization/help/fr_FR/item/rarity.md

```diff
@@ -0,0 +1,5 @@
+La rareté n'affecte que la couleur du nom de l'item.
+* Commun: Blanc
+* Non commun: Jaune
+* Rare: Cyan
+* Épique: Violet clair
```

Comment:
ou plus communément appelé Rose

Reply:
No because in english, it wasn't "Pink" but "Light purple"